tools/releaser/scripts/version-paths.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/bash
 
 export VERSION_UPDATE_PATHS="../**/go.mod,../**/*.go,../tools/**/*.mustache,../.mockery.yaml,../**/*.md"
+
+# don't change doc files to reduce Preview PR noise
+export VERSION_UPDATE_PATHS_PREVIEW="../**/go.mod,../**/*.go,../tools/**/*.mustache,../.mockery.yaml"
```

Comment:
Original file was created to reuse variable across multiple scripts. In this case variable can be moved to original script.